pagopa/io-app
# feat(Payments): [#178212517] Show information about 3DS verification

## Short description
Whenever the user is referred to the 3rd party webview for 3DS verification, we want to show a text content explaining what is happening and why.

## List of changes proposed in this pull request
- use `ScreenContentHeader` in `PayWebViewModal` to show a the relevant content every time the webview is used
- add a new section for localization under [wallet](https://github.com/pagopa/io-app/pull/3135/files#diff-efc72699d0eeebdb187482254790fa072511fb51631053ca553d372cd476197cR1060)

## How to test
- to be able to observe the message, please set [the timeout in the dev server](https://github.com/pagopa/io-dev-api-server/blob/master/src/routers/payment.ts#L154) to something longer than 1s
- navigate under _Wallet_
- add a new card
- click on _save_
- see the new message in the webview

Please check the screencast for visual feedback.

https://user-images.githubusercontent.com/2094604/121875342-a107af80-cd08-11eb-8c67-948f3b8980ba.mov


## Before

<img width="435" alt="Screenshot 2021-06-14 at 12 06 46" src="https://user-images.githubusercontent.com/2094604/121875691-08bdfa80-cd09-11eb-9feb-84b863a04edd.png">


## After

<img width="435" alt="Screenshot 2021-06-14 at 12 06 56" src="https://user-images.githubusercontent.com/2094604/121875708-0fe50880-cd09-11eb-85d8-8c7150d855f2.png">
